FlurryAdapterForMoPubiOS.podspec

```diff
@@ -0,0 +1,12 @@
+Pod::Spec.new do |s|
+  s.name             = "FlurryAdapterForMoPubiOS"
+  s.version          = "7.2.1"
+  s.summary          = "Flurry iOS Adapter for MoPub"
+  s.homepage         = ""
+  s.license          = ""
```

Comment:
This is something that someone at Flurry needs to fill out if you decide to go forward with these changes.